src/functions/postResult/framework/database/query-builder.ts

```diff
@@ -68,7 +68,7 @@ export const buildTestResultInsert = (
   ];
 
   // Specify that dates should be serialised in UTC.
-  return mysql.format(template, args, false, 'UTC');
+  return mysql.format(template, args);
```

Comment:
Hmm, this upgrade will have an impact on the timestamp of the test by the looks of it as per with the unit test here `src/functions/postResult/framework/database/__tests__/query-builder.spec.ts`.

Do we need to update the SQL to do this if the package no longer accepts a timezone?